hammett/conf/global_settings.py

```diff
@@ -12,7 +12,29 @@
 
 HTML_PARSE_MODE = True
 
-LOGGING: dict[str, 'Any'] = {}
+LOGGING: dict[str, 'Any'] =  {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'formatters': {
+        'standard': {
+            'format': '{levelname}: {name}: {asctime}: {message}',
+            'style': '{',
+        },
+    },
+    'handlers': {
+        'console': {
+            'formatter': 'standard',
+            'level': 'DEBUG',
+            'class': 'logging.StreamHandler',
+        },
+    },
+    'loggers': {
+        'hammett': {
+            'handlers': ['console'],
+            'level': 'DEBUG',
```

Comment:
I don't like the idea that level is DEBUG by default. If user need to see debug info he can override settings in his own settings file. I think we dont need to touch this setting